src/components/TopList/index.tsx

```diff
@@ -70,10 +70,7 @@ export const TopList = ({ aggregatedPoolData }: TopListProps) => {
           </div>
           <div className="TopList__pool__row">
             {topPools.map((pool) => (
-              <div
-                className="TopList__pool__row__item"
-                key={getPoolName(pool.assetCodes)}
-              >
+              <div className="TopList__pool__row__item" key={pool.id}>
```

Comment:
unrelated to other things in this PR - this is was throwing a warning bc of multiple pools with the same name

Reply:
Interesting, I thought there could only be a single pool pair.